embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/SessionService.kt

```diff
@@ -9,25 +9,25 @@ internal interface SessionService {
     /**
      * Starts a session in response to a state event.
      */
-    fun startSessionWithState(coldStart: Boolean, timestamp: Long)
+    fun startSessionWithState(coldStart: Boolean, timestamp: Long): String
 
     /**
-     * Ends a session in response to a state event.
+     * Starts a session manually.
      */
-    fun endSessionWithState(timestamp: Long)
+    fun startSessionWithManual(): String
```

Comment:
`startSessionManually()`?

Reply:
I was on the fence about this initially. I'll leave as-is for now as I'm hoping to get to the point where there is an enum passed to the function such as `startSession(StartType)`